lib/Conch/Plugin/AuthHelpers.pm

```diff
@@ -30,17 +30,13 @@ workspace
 			my ( $c, $role_name ) = @_;
 			return 0 unless $c->stash('user_id');
 
-			my $ws = Conch::Model::Workspace->new->lookup_by_name('GLOBAL');
-			return 0 unless $ws;
+			# FIXME: currently does an exact match. should we return true
```

Comment:
We need the ability to look up exact matches of permissions. If we want fuzzy matches, we can add a different helper for that.

Reply:
(No change needed. Just a comment)

Reply:
I was writing helpers for that last week (e.g. `has_at_least_rw($workspace)`) but decided to hold off to see what we wanted to do with the changes with cascading perms.